.prettierrc.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
     {
       files: "*.json",
       options: {
-        tabWidth: 4,
+        tabWidth: 2,
```

Comment:
nit: is this part of our saved-views change? 🤔

Reply:
regardless totally cool

Reply:
not part of saved views, but I'm tired of always having the file reformat back and forth and making diffs longer than they actually are so I changed it